Essentials/test/com/earth2me/essentials/WarpTest.java

```diff
@@ -0,0 +1,98 @@
+package com.earth2me.essentials;
+
+import com.earth2me.essentials.api.IWarps;
+import com.earth2me.essentials.commands.Commandsetwarp;
+import com.earth2me.essentials.commands.WarpNotFoundException;
+import net.ess3.api.IEssentials;
+import net.ess3.api.InvalidWorldException;
+import org.bukkit.Location;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import static org.mockito.Mockito.*;
+
+public class WarpTest {
```

Comment:
For clarification, I've tested these changes manually as well, with LuckyPerms, if that matters.

Reply:
LuckPerms ;)